### DISCUSSION TITLE
my zephyrus G16 CPU power limit not applied

### DISCUSSION THREAD

**Discussion by hoboomon** [seerge/g-helper#667] [Q&A]
<img width="1280" alt="slient" src="https://github.com/seerge/g-helper/assets/113664859/9c9c649c-103d-460c-8fca-b2cce50f5c84">

<img width="1280" alt="performence" src="https://github.com/seerge/g-helper/assets/113664859/ca34e35e-2847-4de1-832d-c987074d2260">

<img width="1280" alt="turbo" src="https://github.com/seerge/g-helper/assets/113664859/465b7d10-c386-4e0b-b6c3-b1a6d0545a6b">

How can I apply the cpu power limit?

Zephyrus G16
i7-12700H
RTX4060 model

**Comment by MckenzieDavis** [comment]
I think this feature only applies for certain AMD processor, correct me if I'm wrong

**Reply by seerge** [reply]
@hoboomon thats not how asus limits work on intell, just run a load in cinebench23 for example and you should see power consumption being within set values

**Reply by hoboomon** [reply]
Is it normal that it doesn't work then? thanks!

**Comment by seerge** [comment]
@hoboomon on a side note, it could be that your specific model falls into Intel CPU limiting issue mentioned in this topic https://github.com/seerge/g-helper/issues/405

Can you try following
1. Close g-helper app
2. Open ``%appdata%/ghelper/config.json`` and add a following line to the top of the file (incl comma) : ```"manual_mode" : 1,```
3. Run g-helper again

And then check if it has effect on real power consumption (under load) ?

**Reply by hoboomon** [reply]
oh Thank you, developer :)

**Reply by hoboomon** [reply]
it works thank you

**Reply by seerge** [reply]
@hoboomon ok, then I will add your model to the list of devices where this approach is needed (so it will work w/o changing config)

**Reply by seerge** [reply]
Do you know if all G16 behave same? or is it only lower-end model that you have with intel 12700?

**Reply by hoboomon** [reply]
I don't know, but in Korea, where I live, only two models are sold: 12700h and 13900h.
